### ISSUE TITLE
Recent update - unable to spawn ./run - waiting 10 seconds

### ISSUE THREAD

**Issue by the-lazy-fox** [filebrowser/filebrowser#1736]
**Description**
Hi,
For few days, following a recent update of the docker image, I'm running this issue.
Impossible to launch it anymore.

**Expected behaviour**
Filebrowser should start smoothly with the yaml file that I always had.

**What is happening instead?**
s6-supervise (child): fatal: unable to exec run: Permission denied
s6-supervise filebrowser: warning: unable to spawn ./run - waiting 10 seconds

**Additional context**
```
version: "3.7"
services:
  filebrowser:
    container_name: filebrowser
    image: filebrowser/filebrowser:latest
    restart: unless-stopped
    environment:
      - PUID=1026
      - PGID=100
      - TZ=Europe/Zurich
    user: "1026:1000"
    ports:
      - 8084:1050
    volumes:
      - /mnt:/srv
      - /home/thelazyfox/docker/filebrowser/config/database.db:/database.db
      - /home/thelazyfox/docker/filebrowser/config/.filebrowser.json:/.filebrowser.json
    labels:
      - com.centurylinklabs.watchtower.enable=true
      - deunhealth.restart.on.unhealthy=true
    healthcheck:
      test: ["CMD-SHELL", "curl --fail http://localhost:1050/ || exit 1"]
      interval: 1m30s
      timeout: 15s
      retries: 3
```

**Comment by methatronc** [comment]
Hello, as I wrote it [in another issue](https://github.com/filebrowser/filebrowser/issues/1737), I believe the filebrowser.json should not be mounted as a hidden file, could you try running your yaml but with 
`- /home/thelazyfox/docker/filebrowser/config/filebrowser.json:/filebrowser.json`
instead ?

**Comment by tmladek** [comment]
Confirming same issue, my docker-compose here (i.e. without `filebrowser.json` as a "hidden file")
```
version: '3'

services:
  service:
    image: filebrowser/filebrowser
    restart: unless-stopped
    volumes:
      - /srv/public:/srv
      - filebrowser_conf:/config
    ports:
      - 8888:80
    command: --database /config/filebrowser.db
    user: 1001:1001
      
volumes:
  filebrowser_conf:
```

**Comment by o1egl** [comment]
Fixed in v2.20.1

**Comment by jayagami** [comment]
Still have this problem, disappeared when commenting user.
